src/daemon/http/server.rs

```diff
@@ -1285,6 +1261,61 @@ async fn api_ca_rta_oneoff(req: Request, path: &mut RequestPath, ca: Handle) ->
     }
 }
 
+//------------ Support acting as a testbed -------------------------------------
+
+async fn testbed(req: Request) -> RoutingResult {
+    if !CONFIG.testbed_enabled {
+        render_not_found(req).await
+    } else {
+        let mut path = req.path().clone();
+        match path.next() {
+            Some("enabled")    => testbed_enabled(req).await,
+            Some("children")   => testbed_children(req, &mut path).await,
+            Some("publishers") => testbed_publishers(req, &mut path).await,
+            _                  => render_not_found(req).await,
```

Comment:
The api uses `render_unknown_method()` whenever the URI maps to a specific part of the API, but there is no method to map to. As opposed to `render_unknown_resource()` when the path is okay but the actual resource at that location cannot be found (e.g. a CA does not exist). The first results in a 400 response, the second in a 404 with a json error, whereas `render_not_found(req)` results in a 404 with HTML which is intended to be browser user facing.

Whether this is all the best way of doing things is of course debatable, but we should still use `render_unknown_method()` here to be consistent.